dotCMS/src/main/java/com/dotcms/rest/api/v1/browser/BrowserQueryForm.java

```diff
@@ -0,0 +1,262 @@
+        return offset;
+    }
+
+    public int getMaxResults() {
+        return maxResults;
+    }
+
+    public boolean isShowWorking() {
+        return showWorking;
+    }
+
+    public boolean isShowArchived() {
+        return showArchived;
+    }
+
+    public boolean isShowFolders() {
+        return showFolders;
+    }
+
+    public boolean isShowFiles() {
+        return showFiles;
+    }
+
+    public boolean isShowPages() {
+        return showPages;
+    }
+
+    public boolean isSortByDesc() {
+        return sortByDesc;
+    }
+
+    public boolean isShowLinks() {
+        return showLinks;
+    }
+
+    public boolean isShowDotAssets() {
+        return showDotAssets;
+    }
+
+    public long getLanguageId() {
+        return languageId;
+    }
+
+    public List<String> getExtensions() {
+        return extensions;
+    }
+
+    public List<String> getMimeTypes() {
+        return mimeTypes;
+    }
+
+    @Override
+    public String toString() {
+        return "BrowserQueryForm{" +
+                "hostFolderId='" + hostFolderId + '\'' +
+                ", filter='" + filter + '\'' +
+                ", sortBy='" + sortBy + '\'' +
+                ", offset=" + offset +
+                ", maxResults=" + maxResults +
+                ", showWorking=" + showWorking +
+                ", showArchived=" + showArchived +
+                ", showFolders=" + showFolders +
+                ", showFiles=" + showFiles +
+                ", showPages=" + showPages +
+                ", sortByDesc=" + sortByDesc +
+                ", showLinks=" + showLinks +
+                ", showDotAssets=" + showDotAssets +
+                ", languageId=" + languageId +
+                ", extensions=" + extensions +
+                ", mimeTypes=" + mimeTypes +
+                '}';
+    }
+
+    public static final class Builder {
+
+        @JsonProperty
+        private String hostFolderId   = Folder.SYSTEM_FOLDER;
+
+        @JsonProperty
+        private String filter         = null;
+
+        @JsonProperty
+        private List<String> mimeTypes  = new ArrayList<>();
+
+        @JsonProperty
+        private List<String> extensions = new ArrayList<>();
+
+        @JsonProperty
+        private String sortBy         = "moddate";
+
+        @JsonProperty
+        private int offset            = 0;
+
+        @JsonProperty
+        private int maxResults        = 100;
+
+        @JsonProperty
+        private boolean showWorking   = true;
+
+        @JsonProperty
+        private boolean showArchived  = false;
+
+        @JsonProperty
+        private boolean showFolders   = false;
+
+        @JsonProperty
+        private boolean showFiles     = false;
+
+        @JsonProperty
+        private boolean sortByDesc    = false;
+
+        @JsonProperty
+        private boolean showLinks     = false;
+
+        @JsonProperty
+        private boolean showPages     = false;
+
+        @JsonProperty
+        private boolean showDotAssets = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using redundant field initializer for 'showDotAssets'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)